__tests__/src/components/WorkspaceMenu.test.js

```diff
@@ -5,9 +5,14 @@ import WorkspaceMenu from '../../../src/components/WorkspaceMenu';
 describe('WorkspaceMenu', () => {
   let wrapper;
   let handleClose;
+  let showZoomControls;
+  let toggleZoomControls;
   beforeEach(() => {
     handleClose = jest.fn();
-    wrapper = shallow(<WorkspaceMenu handleClose={handleClose} />);
+    toggleZoomControls = jest.fn();
+    wrapper = shallow(
+      <WorkspaceMenu handleClose={handleClose} showZoomControls={showZoomControls} />,
```

Comment:
```suggestion
      <WorkspaceMenu
        handleClose={handleClose}
        showZoomControls={showZoomControls}
        toggleZoomControls={toggleZoomControls}
      />,
```

Reply:
I think you just need to wire up the action through to the element, like ☝️

Reply:
Yep, that worked and that makes sense. 👍 Thank you!